org.OpenT2T.Sample.SuperPopular.Helpers/js/index.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+var https = require('https');
+var q = require('q');
+
+//this is our base, we refactor these options in each method
+ var options = 
```

Comment:
Or if you want to have a shared options object that you use to initialize the options for each call, use something like [node-clone](https://github.com/pvorb/node-clone) to make a copy before modifying it to be specific to each call.